src/scos_tekrsa/hardware/tekrsa_sigan.py

```diff
@@ -1,12 +1,8 @@
 import logging
-import time
 
 from scos_actions import utils
-from scos_actions.hardware.hardware_configuration_exception import (
-    HardwareConfigurationException,
-)
+from scos_actions.calibration import sensor_calibration
```

Comment:
I think this is imported in the base sigan_iface

Reply:
I updated this to import from `sigan_iface` instead of from `calibration`.

I thought this broke something so I added some debug messages and better handling for the negative/positive conversion of attenuation values. It turned out that I somehow was using a bad sigan_calibration.json file for testing, and there actually was no issue, but I'll keep those other changes in.